source/Halibut.CertificateGenerator/Halibut.CertificateGenerator.csproj

```diff
@@ -2,24 +2,29 @@
 
   <PropertyGroup>
     <VersionPrefix>0.0.0</VersionPrefix>
-    <TargetFramework>net45</TargetFramework>
     <AssemblyName>Halibut.CertificateGenerator</AssemblyName>
     <OutputType>Exe</OutputType>
     <PackageId>Halibut.CertificateGenerator</PackageId>
     <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
   </PropertyGroup>
 
+  <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
+	  <TargetFrameworks>net472</TargetFrameworks>
```

Comment:
Do we actually need this target at all, netcoreapp2.2 works on Windows?

Reply:
This code won't work under netcoreapp because of a dependency on `CryptoKeySecurity` and I can't justify the time to find alternatives. So the `netcoreapp` build will output 

```
throw new NotSupportedException("Please refer to the README for alternatives that will run on this platform");
```

Reply:
In addition, most of the samples have self signed certificates embedded in them anyway